build/common-vars-template.yml

```diff
@@ -0,0 +1,48 @@
+variables:
+  - name: region
+    value: "{{ .Project.Cloud.Region }}"
+  - name: company
+    value: "{{ .Input.Business.Company }}"
+  - name: project
+    value: "{{ .Project.Name }}"
+  - name: domain
+    value: "{{ .Input.Business.Domain }}"
+  - name: cloud_providers
+    value: "{{ .Input.Cloud.Platform }}"
+  # Terraform Configuration
+
+  # State configuration
+
+  # Stacks operates Terraform states based on workspaces **IT IS VERY IMPORTANT** that you ensure a unique name for each application definition
+  # Furthermore **IT IS VERY IMPORTANT** that you change the name of a workspace for each deployment stage
+  # there are some best practices around this if you are going for feature based environments
+  # avoid running anything past dev that is not on master
+  - name: tf_state_rg
+    value: "{{ .Input.Terraform.Backend.Group }}"
+  - name: tf_state_storage
+    value: "{{ .Input.Terraform.Backend.Storage }}"
+  - name: tf_state_container
+    value: "{{ .Input.Terraform.Backend.Container }}"
+
+  # App Specific Taxonomy
+  - name: component
+    value: de  # core
```

Comment:
In the Stacks CLI I believe there's a value `.Input.Business.Component` - I think it makes sense to use that here (and then move under variables)

Reply:
sure